lemminx-liberty/src/main/java/io/openliberty/lemminx/liberty/services/FeatureService.java

```diff
@@ -35,9 +38,13 @@ public static FeatureService getInstance() {
 
   // Cache of liberty version -> list of supported features
   private Map<String, List<Feature>> featureCache;
+  private List<Feature> defaultFeatureList;
+  private long featureUpdateTime;
 
   private FeatureService() {
     featureCache = new HashMap<>();
+    defaultFeatureList = new ArrayList<>();
```

Comment:
I think we can just leave this as null until we initialize defaultFeatureList. And in getDefaultFeatureList change the check from `isEmpty` to `== null`.